### PR TITLE
SPM checkpoint - Storage qs runs

### PR DESCRIPTION
**Pull request by paulb777**

### CODE REVIEW

**Comment by paulb777** [comment]
Storage quickstart now builds and runs with Auth.  Only remaining qs workaround is the Firebase module.

**Comment by paulb777** [comment]
Now the Firebase module works and the FirebaseStorage quickstart runs without alterations.

File: GoogleUtilities/Reachability/GULReachabilityChecker.m [hunk 1]
```diff
@@ -14,12 +14,17 @@
 
 #import <Foundation/Foundation.h>
 
-#import "GULReachabilityChecker+Internal.h"
-#import "Private/GULReachabilityChecker.h"
-#import "Private/GULReachabilityMessageCode.h"
-
+#import "Reachability/GULReachabilityChecker+Internal.h"
```
Comment (ryanwilson) [inline GoogleUtilities/Reachability/GULReachabilityChecker.m]: What's the reason for these changing to have the root path?

Comment (paulb777) [reply GoogleUtilities/Reachability/GULReachabilityChecker.m]: Partly to minimize the number of settings in the Package.swift and partly to experiment. 

There will need to be a thorough review and possible refactor of all imports before merging to master.

**Pull request merged**